src/Confluent.Kafka/Producer.cs

```diff
@@ -241,19 +257,19 @@ public void HandleDeliveryReport(Message message)
         ///     Initializes a new Producer instance.
         /// </summary>
         /// <param name="config">
-        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
-        ///     TODO: Link to confluent-kafka-dotnet page with dotnet specific parameters also (i.e. default.topic.config).
+        ///     librdkafka configuration parameters (refer to https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md).
+        ///     Topic configuration parameters are specified via the "default.topic.config" sub-dictionary config parameter.
         /// </param>
         /// <param name="manualPoll">
         ///     If true, does not start a dedicated polling thread to trigger events or receive delivery reports -
-        ///     you must call the Poll method periodically instead.
+        ///     you must call the Poll method periodically instead. Typically you should set this parameter to false.
         /// </param>
         /// <param name="disableDeliveryReports">
-        ///     If true, disables notification of delivery reports. Note: if set to true and you use a ProduceAsync variant that return
-        ///     a Task, the Tasks will never complete. Generally you should leave this parameter as false. Set it to true for "fire and
+        ///     If true, disables delivery report notification. Note: if set to true and you use a ProduceAsync variant that returns
+        ///     a Task, the Tasks will never complete. Typically you should set this parameter to false. Set it to true for "fire and
```

Comment:
I'll mark as unstable (you've probably noticed i like doing that).

BTW: unrelated: I've got another idea as to how to factor the general client functionality currently exposed on producers and consumers (and marked as unstable). you're going to love it. This time I actually like the idea too.